metadata-ingestion/src/datahub/ingestion/run/pipeline.py

```diff
@@ -384,11 +387,21 @@ def run(self) -> None:
                 except SystemExit:
                     raise
                 except Exception as e:
-                    # TODO: Transformer errors should cause the pipeline to fail.
                     logger.error(
                         "Failed to process some records. Continuing.", exc_info=e
                     )
 
+                    try:
+                        if telemetry.telemetry_instance.sentry_enabled:
+                            import sentry_sdk
+
+                            sentry_sdk.capture_exception(e)
```

Comment:
ideally all `import sentry_sdk` code should live in telemetry.py - can we extract this into a helper method (including the try catch - the helper should never throw)?

Reply:
or even better - we have a contextlib so that the client does `with telemetry_instance.log_exceptions_in_sentry():`

but not required